src/Update/Updates.php

```diff
@@ -551,5 +551,111 @@ public function update_9001000() {
     $this->updater->getOutput()->writeln("");
     $this->updater->getOutput()->writeln($formattedBlock);
     $this->updater->getOutput()->writeln("");
+
+    $this->updater->syncWithTemplate('blt/ci.blt.yml', TRUE);
+    $messages = ['blt/ci.blt.yml has been updated. Review it for any custom changes that may have been overwritten.'];
+
+    $formattedBlock = $this->updater->getFormatter()->formatBlock($messages, 'ice');
+    $this->updater->getOutput()->writeln("");
+    $this->updater->getOutput()->writeln($formattedBlock);
+    $this->updater->getOutput()->writeln("");
+
+    // Update composer.json to include new BLT required/suggested files.
+    // Pulls in wikimedia/composer-merge-plugin and composer/installers settings.
+    $project_composer_json = $this->updater->getRepoRoot() . '/composer.json';
+    $template_composer_json = $this->updater->getBltRoot() . '/template/composer.json';
+    $munged_json = ComposerMunge::mungeFiles($project_composer_json, $template_composer_json);
+    $bytes = file_put_contents($project_composer_json, $munged_json);
+    if (!$bytes) {
+      $messages = ["Could not update $project_composer_json."];
+    }
+    else {
+      $messages = ["Updated $project_composer_json. Review changes, then re-run composer update."];
+    }
+
+    $formattedBlock = $this->updater->getFormatter()->formatBlock($messages, 'ice');
+    $this->updater->getOutput()->writeln("");
+    $this->updater->getOutput()->writeln($formattedBlock);
+    $this->updater->getOutput()->writeln("");
+  }
+
+  /**
+   * 9.1.1.
+   *
+   * @Update(
+   *    version = "9001001",
+   *    description = "Add support for running Drush 8 commands in BLT and Acquia Cloud."
+   * )
+   */
+  public function update_9001001() {
+
+    // Sync updates to drush 8 drushrc.php in case it has been removed or added to ignore-existing.txt.
+    $drushrcFile = 'drush/drushrc.php';
+    $this->updater->syncWithTemplate($drushrcFile, TRUE);
+
+    $messages[] = "drushrc.php added/updated to provide Drush 8 command support";
+
+    // Ensure drush and other problematic packages are not in root composer.json
+
+    $composer_json = $this->updater->getComposerJson();
+    $remove_packages = [
+      'drush/drush',
+    ];
+    foreach ($remove_packages as $package) {
+      unset($composer_json['require'][$package]);
+      unset($composer_json['require-dev'][$package]);
+    }
+    $this->updater->writeComposerJson($composer_json);
+
+    $messages[] = "Drush removed from root composer config";
+
+    // Ensure local drush 9 in vendor has been removed. 
+
+    $this->updater->getFileSystem()->remove('vendor/drush');
```

Comment:
Seems like we'd be better off running `blt:drush:redispatch`, which removes additional, related files.